src/main/java/uk/gov/register/views/ViewFactory.java

```diff
@@ -27,17 +29,19 @@
     private final Provider<HomepageContentConfiguration> homepageContentConfiguration;
     private final Provider<RegisterLinkService> registerLinkService;
     private final ItemConverter itemConverter;
+    private final Provider<RegisterName> registerNameProvider;
 
     @Inject
-    public ViewFactory(RequestContext requestContext,
-                       PublicBodiesConfiguration publicBodiesConfiguration,
-                       GovukOrganisationClient organisationClient,
-                       RegisterDomainConfiguration registerDomainConfiguration,
-                       Provider<HomepageContentConfiguration> homepageContentConfiguration,
-                       Provider<RegisterTrackingConfiguration> registerTrackingConfiguration,
-                       RegisterResolver registerResolver,
-                       Provider<RegisterReadOnly> register,
-                       Provider<RegisterLinkService> registerLinkService, ItemConverter itemConverter) {
+    public ViewFactory(final RequestContext requestContext,
+                       final PublicBodiesConfiguration publicBodiesConfiguration,
+                       final GovukOrganisationClient organisationClient,
+                       final RegisterDomainConfiguration registerDomainConfiguration,
+                       final Provider<HomepageContentConfiguration> homepageContentConfiguration,
+                       final Provider<RegisterTrackingConfiguration> registerTrackingConfiguration,
+                       final RegisterResolver registerResolver,
+                       final Provider<RegisterReadOnly> register,
+                       final Provider<RegisterLinkService> registerLinkService, final ItemConverter itemConverter,
+                       final Provider<RegisterName> registerNameProvider) {
```

Comment:
`RegisterName` is available from `RegisterReadOnly` - is this because you need it inside of a `Provider`?

Reply:
The provider is needed in the different TurtleWriter constructors